openregistry/concierge/worker.py

```diff
@@ -150,7 +150,9 @@ def check_assets(self, lot, status='pending'):
             except RequestFailed as e:
                 logger.error('Falied to get asset {0}. Status code: {1}'.format(asset_id, e.status_code))
                 raise RequestFailed('Failed to get assets')
-            if asset.status != status or asset.relatedLot != lot['id']:
```

Comment:
`if ('relatedLot' in asset and asset.relatedLot != lot['id']) or asset.status != status:`